bin/scoredata.js

```diff
@@ -1,290 +1,274 @@
 #!/usr/bin/env node --harmony
 
-var program = require('commander');
-var Excel = require('exceljs');
-var JSONStream = require('JSONStream');
-var fs = require('fs');
-var chalk = require('chalk');
-var ss = require('simple-statistics');
-var path = require('path');
+/* eslint-disable no-console */
+
+const program = require('commander');
+const Excel = require('exceljs');
+const fs = require('fs');
+const chalk = require('chalk');
+const ss = require('simple-statistics');
+const path = require('path');
 
 const MAX_EGV_PER_DAY = 288;
 const DAY_IN_MILLI = 86400000;
```

Comment:
this is also exported in `lib` if you want to rationalise

Reply:
They're not in `lib`. They're command-line imports, so they're fine here. We wouldn't want to import them for JS-only use.